internal/provider/protonvpn/updater/api.go

```diff
@@ -23,6 +23,7 @@ type logicalServer struct {
 	Region      *string          `json:"Region"`
 	City        *string          `json:"City"`
 	Servers     []physicalServer `json:"Servers"`
+	Features    int              `json:"Features"`
```

Comment:
nit I guess these are positive only numbers, so let's use uint

```suggestion
	Features    uint              `json:"Features"`
```

Or maybe event `uint16` I think? 🤔

Reply:
Not familiar enough with Go 😅. Both uint16 and uint sound fine to me, with uint having the advantage of not having to be changed if a feature is added and uint16 not reserving unneeded memory space (if I understand correctly). Let me know which of the 2 has your preference!

Reply:
`uint` has the advantage to reject negative inputs and have compilation safety against negative numbers.... just in case ™️ 

`uint16` restricts it further (from 64 or 32 bits to 16 bits on all platforms), and perhaps highlight more it's used as a bit array and not just a number. Let's try uint16 😉